addons/account/static/src/components/bills_upload/bills_upload.scss

```diff
@@ -21,7 +21,7 @@
 .o_widget_account_file_uploader {
   .btn-primary.oe_kanban_action_button {
     a {
-      color: $white;
```

Comment:
I don't understand why we place a link inside a button, could you investigate?
If it's not possible to fix it, can we eventually add a `text-reset` class instead?

Reply:
Found it !

I turned that link into a span and it seems to work fine, let's see how tech review will go !

Reply:
Replacing `<a/>` with `<span/>` removes the ability to select the button using the keyboard.

Perhaps moving the btn classes from `<div/>` to `<a/>` might work. To be checked.